src/main/appender.cpp

```diff
@@ -375,6 +411,65 @@ void Appender::FlushInternal(ColumnDataCollection &collection) {
 	context->Append(*description, collection);
 }
 
+void Appender::AppendDefaultsToVector(Vector &result, idx_t vector_size, idx_t column, SelectionVector &sel,
+                                      idx_t count) {
+	sel.Verify(count, vector_size);
```

Comment:
Since we need to simplify `AppendDefault` to basically just add a index to a selection vector anyway (or use a pre-computed default value) - I would vote for removing this function